### PR TITLE
Add axios-tauri-adapter

### PR DESCRIPTION
**Pull request by KaKi87**
- [x] **I have read the [contributing guidelines](contributing.md).**

- [x] Title as described.
- [x] Added to the right category.
- [x] Added to the end of a list.
- [x] The description of your item is a sentence with less than 24 words.
- [x] No links and parentheses in description. 
- [x] The description starts with a capital and ends with a full stop/period.
- [x] The description doesn't start with `A` or `An`.
- [x] No unnecessary words already provided in the context (e.g. `for Tauri`, `a Tauri plugin`).
- [x] Uses proper case for terms (e.g. `GitHub`, `TypeScript`, `ESLint`, etc.)
- [x] When mentioning tools, omits versions whenever possible (e.g. use `TypeScript` instead of `TypeScript 4.x`.
- [x] When mentioning package names, uses quotes whenever possible.

### Plugins/Tools

- [x] The plugin/tool is working with **Tauri 1.x and onward**.
- [x] The project is Open Source.
- [x] The repo should be at least 30 days old.
- [x] The documentation is in English.
- [x] The project is active and maintained (inactive projects for longer 6 months will be removed without further notice).
- [x] The project accepts contributions.
- [x] Not a commercial product.

### Starter Templates

- [ ] The starter template is working with **Tauri 1.x and onward**.
- [ ] The documentation is in English.
- [ ] The starter template most provide enough instructions / documentation about how to start up and what's is included.
- [ ] The repo should be at least 30 days old.
- [ ] Should be differentiable from the existing starter templates.

### Apps

- [ ] The project is Open Source.
- [ ] The readme is in English.
- [ ] The app is original and not too simple.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ This is where we collect all of the best stuff from the ecosystem and community.
 
 - [vue-cli-plugin-tauri](https://github.com/tauri-apps/vue-cli-plugin-tauri) ![official](https://img.shields.io/badge/-official-FFC131) - Turn your Vue SPA into a lightweight cross-platform desktop app.
 - [vite-plugin-tauri](https://github.com/amrbashir/vite-plugin-tauri) - Integrate Tauri in a Vite project to build cross-platform apps.
+- [axios-tauri-adapter](https://git.kaki87.net/KaKi87/axios-tauri-adapter) - Axios adapter for the `http` module.
```
Comment (amrbashir) [inline README.md]:
```suggestion
- [axios-tauri-adapter](https://git.kaki87.net/KaKi87/axios-tauri-adapter) - Axios adapter for the `@tauri-apps/api/http` module.
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KaKi87) [reply README.md]: Done : 900faf9

**Comment by JonasKruckenberg** [comment]
Interesting git provider choice.
No judgment, just never seen that before.

**Review by JonasKruckenberg — APPROVED** [review]

**Comment by KaKi87** [comment]
> Interesting git provider choice. No judging, just never seen that before.

Yes, I'm providing myself with a self-hosted instance of [Gitea](https://gitea.io/en-us/).

**Comment by JonasKruckenberg** [comment]
> Yes, I'm providing myself with a self-hosted instance of [Gitea](https://gitea.io/en-us/).

Neat! It's not super mobile friendly but still

**Comment by KaKi87** [comment]
> Neat!

:+1: 

> It's not super mobile friendly

Agreed, that's why someone created [GitNex](https://gitnex.com/) for Android.

**Review by amrbashir — APPROVED** [review]

**Pull request merged**